setup.cfg

```diff
@@ -19,12 +19,18 @@ setup_requires =
     setuptools >= 38.3.0
     setuptools_scm
 install_requires =
+    cartopy
     cftime == 1.0.1
+    cf_units
     clize
+    dask
+    netCDF4
     numpy
-    sigtools
+    python-dateutil
+    pytz
```

Comment:
I wonder if this can be optional since it seems to only be used for generating ancillary files. (no worries if not)

Reply:
Also why is the version pinned in the environment yaml file but not here..?

Reply:
My thinking was that any package directly imported in IMPROVER should be included. When I say "directly imported" I mean without using, e.g., importlib as is the case for numba and the other optional dependencies.

Reply:
I've unpinned it now in the Python 3.7 environment file.